lib/src/utils/utils.dart

```diff
@@ -93,3 +94,24 @@ Map<String, T>? removeAliasPrefix<T>(Map<String, T?>? map) {
   });
   return mapResult;
 }
+
+/// Generate a random seed
+String generateRandomSeed() {
```

Comment:
Where are you going to use this method ? From the wallet ?

Reply:
`MessagingService` class